<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateCompDialog</class>
 <widget class="QDialog" name="CreateCompDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>650</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Create competition</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#CreateCompDialog {
	background: url(:/settings_background.png)  no-repeat center center fixed;
}</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>550</y>
     <width>541</width>
     <height>54</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="lauoutButtons">
    <item>
     <widget class="QPushButton" name="btnMenu">
      <property name="styleSheet">
       <string notr="true">QPushButton { 
	border: 1px solid #6d6d6d transparent;
	background: qlineargradient(x1: 0, y1: 1, x2: 0, y2: 0,  stop: 0 #d9b900, stop: 1 #ffd800);
	border-radius: 5px;
	color: #3a3a3a;
	font-size: 20px;
	height: 50px;
	width: 250px;
}

QPushButton:hover { 
	background-color: #ffd800;
}

QPushButton:pressed {
	background-color: #d9b900;
}</string>
      </property>
      <property name="text">
       <string>Menu</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="btnCreate">
      <property name="styleSheet">
       <string notr="true">QPushButton { 
	border: 1px solid #6d6d6d transparent;
	background: qlineargradient(x1: 0, y1: 1, x2: 0, y2: 0,  stop: 0 #d9b900, stop: 1 #ffd800);
	border-radius: 5px;
	color: #3a3a3a;
	font-size: 20px;
	height: 50px;
	width: 250px;
}

QPushButton:hover { 
	background-color: #ffd800;
}

QPushButton:pressed {
	background-color: #d9b900;
}</string>
      </property>
      <property name="text">
       <string>Create</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>50</y>
     <width>600</width>
     <height>471</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#frame {
	border: 0px;
	background-color: rgba(0, 0, 0, 0.6);
	border-radius: 15px;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="lblSettings">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>601</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font-size: 24px;
width: 240px;
font-weight: bold;</string>
    </property>
    <property name="text">
     <string>Create new competition:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lblPreferences">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>117</width>
      <height>37</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font-size: 18px;
width: 240px;</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>Preferences:</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="margin">
     <number>8</number>
    </property>
   </widget>
   <widget class="QLabel" name="lblNumberOfTeams">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>161</width>
      <height>37</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font-size: 18px;
width: 240px;</string>
    </property>
    <property name="text">
     <string>Number or teams:</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="margin">
     <number>8</number>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>180</y>
      <width>250</width>
      <height>35</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="title">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <widget class="QRadioButton" name="rbtn32">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>0</y>
       <width>115</width>
       <height>35</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: none;</string>
     </property>
     <property name="text">
      <string>32</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbtn16">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>115</width>
       <height>35</height>
      </rect>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::DefaultContextMenu</enum>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>16</string>
     </property>
    </widget>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>230</y>
      <width>250</width>
      <height>101</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Russia</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Portugal</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Austria</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>France</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Germany</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Netherlands</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Spain</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="lblTeams">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>72</width>
      <height>37</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font-size: 18px;
width: 240px;</string>
    </property>
    <property name="text">
     <string>Teams:</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="margin">
     <number>8</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="edtTitle">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>130</y>
      <width>250</width>
      <height>35</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 18px;
width: 240px;
height: 35px;
padding-left: 5px;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lblTitle">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>211</width>
      <height>37</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font-size: 18px;
width: 240px;</string>
    </property>
    <property name="text">
     <string>Title of the competition:</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="margin">
     <number>8</number>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbPreferences">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>80</y>
      <width>250</width>
      <height>35</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 18px;
width: 220px;
height: 35px;
padding-left: 5px;</string>
    </property>
    <property name="duplicatesEnabled">
     <bool>false</bool>
    </property>
    <property name="modelColumn">
     <number>0</number>
    </property>
    <item>
     <property name="text">
      <string>Custom</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>UEFA EURO</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>FIFA World Cup</string>
     </property>
    </item>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>btnMenu</tabstop>
  <tabstop>cmbPreferences</tabstop>
  <tabstop>edtTitle</tabstop>
  <tabstop>rbtn16</tabstop>
  <tabstop>rbtn32</tabstop>
  <tabstop>listWidget</tabstop>
  <tabstop>btnCreate</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
